server/controllers/setting.js

```diff
@@ -0,0 +1,35 @@
+const {Setting} = require('../models/Setting')
+
+exports.GetUserSetting = function(req, res){
+    const username = req.params.username
+  
+    if (!username) {
+      res.status(401).send("Failed to provide a username")
+    }
+  
+    const handleFindUserSetting = function (err, setting){
+      if (err) {
+        console.log(err)
+        res.status(500).send("Problem occurred when getting user setting!");
+      } else if (!setting) {
+        res.status(200).json(defaultSettings)
+      } else {
+        res.status(200).json(setting);
+      }
+    }
+  
+    Setting.findOne({ username: username }, handleFindUserSetting);
+  }
+  
+  exports.updateUserSetting = function(req, res){
```

Comment:
Can you capitalize this to make it consistent with above? So `exports.UpdateUserSettings`